task.sql

```diff
@@ -38,3 +38,17 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset daria.savchenko:5 labels:0.0.2
+CREATE TABLE Users (
+    ID INT,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(100),
+    PRIMARY KEY (ID)
+);
+--rollback DROP TABLE Users;
+
+--changeset daria.savchenko:6 labels:0.0.3
+CREATE INDEX Email ON Users (Email);
+--rollback ALTER TABLE Users DROP INDEX Email;
```

Comment:
Add Eof(End-of-file), need to add blank line at the end
![image](https://github.com/user-attachments/assets/79d8cfc8-2b7d-4ce7-a90d-09601877463c)